src/emulator/functionsEmulator.ts

```diff
@@ -400,20 +436,12 @@ You can probably fix this by running "npm install ${
       persistent: true,
     });
 
-    const diagnosticBundle: FunctionsRuntimeBundle = {
-      cwd: this.functionsDir,
-      projectId: this.projectId,
-      triggerId: "",
-      ports: {},
-      disabled_features: this.args.disabledRuntimeFeatures,
-    };
-
     // TODO(abehaskins): Gracefully handle removal of deleted function definitions
     const loadTriggers = async () => {
-      const runtime = InvokeRuntime(this.nodeBinary, diagnosticBundle);
+      const runtime = InvokeRuntime(this.nodeBinary, this.bundleTemplate);
```

Comment:
Could you add a comment explaining that sending a template-only message is just a "diagnostic" call to set things up?

Reply:
Yeah that's fair